src/kube-runtime/src/plugins/plugin_utils.py

```diff
@@ -67,6 +69,19 @@ def plugin_init():
     return [plugin_config, args.pre_script, args.post_script]
 
 
+def try_to_install_by_cache(group_name):
+    source_folder = '/opt/package_cache'
+    target_folder = os.path.join(PAI_WORK_DIR, 'package_cache')
+    exists_group_names = os.listdir(source_folder)
+    needed_group_names = [name for name in exists_group_names if name.startswith(group_name + '-')]
+    for name in needed_group_names:
+        name_source_folder = os.path.join(source_folder, name)
+        name_target_folder = os.path.join(target_folder, name)
+        if not(os.path.exists(name_target_folder)):  # avoid duplicate copy
+            shutil.copytree(name_source_folder, name_target_folder)
+    return '/bin/bash {}/runtime.d/install_group.sh '.format(PAI_WORK_DIR) + group_name
+
```

Comment:
I think it would the below behavior would be better
```python
def try_to_install_by_cache(group_name: string, failed_cmds: list):
    # no change, copy caches
    cached_cmd = '/bin/bash {}/runtime.d/install_group.sh '.format(PAI_WORK_DIR) + group_name
    return '%s || {%s}' % (cached_cmd, ';'.join(failed_cmds))
```
and the example usage is 
```bash
cmds = [
    ...
    try_to_install_by_cache('ssh', ['apt-get update', 'apt-get install -y openssh-client openssh-server']),
]
```

Reply:
will fix this in a different PR